apps/ledger-live-desktop/src/renderer/families/mina/StakeBanner.tsx

```diff
@@ -0,0 +1,60 @@
+import useFeature from "@ledgerhq/live-common/featureFlags/useFeature";
+import { AccountBanner } from "~/renderer/screens/account/AccountBanner";
+import { track } from "~/renderer/analytics/segment";
+import { stakeDefaultTrack } from "~/renderer/screens/stake/constants";
+import React from "react";
+import { openModal } from "~/renderer/actions/modals";
+import { useDispatch } from "react-redux";
+import { MinaAccount } from "@ledgerhq/live-common/families/mina/types";
+import { useGetStakeLabelLocaleBased } from "~/renderer/hooks/useGetStakeLabelLocaleBased";
+import { useTranslation } from "react-i18next";
+
+const StakeBanner: React.FC<{ account: MinaAccount }> = ({ account }) => {
+  const { t } = useTranslation();
+  const dispatch = useDispatch();
+  const stakeAccountBanner = useFeature("stakeAccountBanner");
+  const cta = useGetStakeLabelLocaleBased();
+
+  if (!stakeAccountBanner?.enabled) return null;
+
+  const hasDelegation = account.resources?.stakingActive;
+
+  const title = hasDelegation ? t("mina.stakeBanner.altTitle") : t("mina.stakeBanner.title");
+
+  const description = hasDelegation
+    ? t("mina.stakeBanner.altDescription")
+    : t("mina.stakeBanner.description");
+
+  const linkText = t("common.learnMore");
+  const linkUrl = "https://www.ledger.com/staking/staking-mina";
```

Comment:
this link can be declared outside of the component, to avoid redeclaring on runtime.

Reply:
Yes agree! Moved it out